MapboxNavigation/TurnArrowView.swift

```diff
@@ -93,6 +93,7 @@ public class TurnArrowView: UIView {
                 flip = true
             case .uTurn:
                 StyleKitArrows.drawArrow180(scale: scale)
+                flip = [.left, .slightLeft, .sharpLeft].contains(ManeuverDirection(angle: angle))
```

Comment:
This method has access to the `step` property in the surrounding class. Check the RouteStep’s `maneuverDirection` instead of creating a new one.